src/index.html

```diff
@@ -11,7 +11,177 @@
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms-and-conditions"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="mail">E-mail:</label>
+          <input
+            type="email"
+            name="mail"
+            id="mail"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="form-field">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+          >
+          <label for="no">No</label>
+        </div>
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              id="color"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="time-for-bed">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time-for-bed"
+              id="time-for-bed"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="car">
+            What are your favorite brand of cars?
+            <select
+              name="car"
+              id="car"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+
+        </div>
+        <div class="form-field">
+          <label for="rate">
+            How do you rate our work?
+            <input
+              type="range"
+              name="rate"
+              id="rate"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info</legend>
+        <div class="form-field">
+          <label for="comment">
+            Comments:
+            <textarea
+              name="comment"
+              id="comment"
+              cols="20" rows="2"
+            >
+            </textarea>
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="recomendation">
+            Would you recommend us?
+            <select
+              name="recomendation"
+              id="recomendation">
```

Comment:
[checklist p3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
            <select
              name="recomendation"
              id="recomendation"
            >
```